packages/@expo/cli/src/start/server/metro/debugging/createDebugMiddleware.ts

```diff
@@ -16,7 +17,11 @@ export function createDebugMiddleware(metroBundler: MetroBundlerDevServer) {
     logger: createLogger(chalk.bold('Debug:')),
     unstable_customInspectorMessageHandler: createHandlersFactory(metroBundler),
     unstable_experiments: {
-      enableOpenDebuggerRedirect: true,
+      // Enable the Network tab in React Native DevTools
+      enableNetworkInspector: true,
+      // Only enable opening the browser version of React Native DevTools when debugging.
+      // This is useful when debugging the React Native DevTools by going to `/open-debugger` in the browser.
+      enableOpenDebuggerRedirect: env.EXPO_DEBUG,
```

Comment:
This is only enabled when running in debug mode. RNDT should primarily be used as-is, and the dev server should not "block" endpoints that could be passed through to the projects.